.env.example

```diff
@@ -42,6 +42,15 @@ MAIL_FROM_NAME=
 MAILGUN_DOMAIN=
 MAILGUN_SECRET=
 
+# Slack Webhook Endpoint path to be appended to https://hooks.slack.com/services/.
+SLACK_ENDPOINT=
+
+# Default slack channel for the bot to post in.
+SLACK_CHANNEL=
+
+# Default username to post as
+SLACK_USERNAME=
```

Comment:
Yes, I agree, it is overkill putting it above the username; more so if I switch to "BOT_USERNAME". 

I don't know what the deal is with changing it (newbie). Do I wait for further comments or just change it?